Source/WebCore/editing/TextIterator.cpp

```diff
@@ -455,7 +464,10 @@ static bool isRendererAccessible(const RenderObject* renderer, TextIteratorBehav
     if (renderer->isSkippedContent()) {
         if (!behaviors.contains(TextIteratorBehavior::EntersSkippedContentRelevantToUser))
             return false;
-        return style.usedContentVisibility() == ContentVisibility::Auto;
+        if (style.usedContentVisibility() == ContentVisibility::Auto)
+            return true;
+        if (renderer->document().settings().detailsAutoExpandEnabled() && renderer->style().autoRevealsWhenFound())
```

Comment:
The code that sets the `RenderStyle` bit already checks the settings, so can we leave out the settings check here?

Reply:
Yeah true, it's not needed here.